Tasks/BelkevichAleksandr/Task4/Task4.py

```diff
@@ -0,0 +1,35 @@
+def check_str():
+    return f'{sum(map(str.isupper, "The quick Brown Fox"))} upper case, {sum(map(str.islower, "The quick Brown Fox"))} lower case'
```

Comment:
oh, some argument would be nice, but I will not nitpick about it

btw the map usage is very good here